README.md

```diff
@@ -473,6 +473,20 @@ All statements above will return false if used with `===`
 
 <p align="right"><a href="#table-of-contents">⬆ Return to top</a></p>
 
+## ![✔] 4.9 Refactor regularly using static analysis tools
+
+**TL;DR:** Using static analysis tools helps by giving objective ways to improve code quality and keep your code maintainable. Keeping out code smells and bloat will make maintenance easier and provides better readability to your code.
```

Comment:
@TheHollidayInn Just an idea - the first sentence perfectly explains the motif. Give some tangible examples instead of the second line (discover complex code, code duplication, untested code paths)

Maybe also remind that it can live within CI and make builds fail or even within IDE and show immediate insights as the developer types